### ISSUE TITLE
Hotkeys aren't truly global, are "consumed" by some apps

### ISSUE THREAD

**Issue by Aran-Fey** [mzomparelli/zVirtualDesktop#67]
Some apps, while focused, prevent the hotkeys from taking effect.
An example is the Administrator Command Prompt. Pressing Ctrl+Alt+1 inserts a ¡ character instead of switching to desktop # 1.

**Comment by mzomparelli** [comment]
It has something to do with being a privileged window. It does not do this when you use a non-admin command prompt. I noticed Task Manager does this as well.

Running zVD as admin helps. Try it and report back please. I was able to use a hotkey while an elevated command prompt had focus when zVD was run as admin.

**Comment by Aran-Fey** [comment]
You're right, running zVD as admin makes everything work as expected.

I'm pretty sure there's a way to make this work without admin privileges though. I've used a different virtual desktop manager in the past and never experienced this problem. I'll use your workaround for now, but if you could make this work without admin privileges, that would be fantastic.

**Comment by mzomparelli** [comment]
The other program you mention may already be running with elevated privileges. Can you confirm that? I don't think this is a work around. This is the expected behavior and is there to safeguard your PC. I could have it run with elevated privileges, but you'll be prompted by Windows for confirmation.

**Comment by Aran-Fey** [comment]
I've checked the other program and it seems to be running without admin privileges. I've checked the Compatibility tab and confirmed that "run as administrator" is unchecked. It doesn't seem to elevate its privileges after starting either, because Windows never shows me any dialogs asking me for permission.

But it's not a big problem **and** there's an easy workaround, so this really isn't a particularly important feature to have. I'm sure you have a lot of other stuff to work on.
